frameworks/Dart/aqueduct/server.dart

```diff
@@ -0,0 +1,227 @@
+import 'dart:async';
+import 'dart:io';
+import 'dart:math';
+
+import 'package:aqueduct/aqueduct.dart';
+import 'package:mustache/mustache.dart' as mustache;
+
+export 'dart:async';
+export 'dart:io';
+export 'package:aqueduct/aqueduct.dart';
+
+final _stripped_text = new ContentType("text", "plain");
+final _stripped_json = new ContentType("application", "json");
+
+final _random = new Random();
+
+// World table size
+const int _world_table_size = 10000;
+// Fortune table size used only for generation of data
+const int _FORTUNE_TABLE_SIZE = 100;
+
+const int _min_query_count = 1;
+const int _max_query_count = 500;
+
+Future main() async {
+  try {
+    var app = new Application<DartAqueductBenchmarkSink>();
+    var config = new ApplicationConfiguration()
+      ..port = 8080
+      ..configurationFilePath = "config.yaml";
```

Comment:
It looks like there is a `config.yaml` and a `postgresql.yaml` with the same contents.  Is only `config.yaml` used?

Reply:
Oh. Yes, only the config.yaml is used. I'll correct this.